### PR TITLE
[home-assistant] Update documentation for HA and Ingress

### PR DESCRIPTION
**Pull request by angryninja48**
Signed-off-by: Jon Baker <jonbaker85@gmail.com>

**Description of the change**

Update documentation for Ingress + Websockets

**Benefits**

Better doco

**Possible drawbacks**

None

**Applicable issues**

- fixes #

**Additional information**

**Checklist** 
- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [ ] Chart is using our [common](https://github.com/k8s-at-home/charts/tree/master/charts/common) library as a dependency.
- [ ] (optional) Variables are documented in the README.md

### CODE REVIEW

**Comment by angryninja48** [comment]
Was unsure about the best way to about this was originally thinking something like this would be easier:

```yaml
    {{- if $.Values.nginx.enablewebsockets }}
    nginx.org/websocket-services: {{ .Values.service.name }}
    {{- end}}
```

**Comment by onedr0p** [comment]
I am surprised this needs to be enabled... I do not use any websocket annotations with ingress-nginx, and it still works.

**Comment by angryninja48** [comment]
Kube
```
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.1", GitCommit:"c4d752765b3bbac2237bf87cf0b1c2e307844666", GitTreeState:"clean", BuildDate:"2020-12-19T07:38:38Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.4", GitCommit:"d360454c9bcd1634cf4cc52d1867af5491dc9c5f", GitTreeState:"clean", BuildDate:"2020-11-11T13:09:17Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}
```

Helm
```
version.BuildInfo{Version:"v3.4.2", GitCommit:"23dd3af5e19a02d4f4baa5b2f242645a1a3af629", GitTreeState:"dirty", GoVersion:"go1.15.5"}
```

Nginx
```
Image: nginx/nginx-ingress:1.9.1
```

HA
```
Image: homeassistant/home-assistant:2020.12.1
```

**Comment by onedr0p** [comment]
I am OK with the documentation you added but you need to bump the chart version :)

**Comment by angryninja48** [comment]
I wrestled with it all yesterday and eventually got it working enabling websockets

**Comment by onedr0p** [comment]
Maybe it's a nginx-ingress vs. ingress-nginx thing :D

File: charts/home-assistant/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 2020.12.1
 description: Home Assistant
 name: home-assistant
-version: 5.2.1
+version: 5.3.1
```
Comment (onedr0p) [inline charts/home-assistant/Chart.yaml:5]: this is not a feature, please bump it to 5.2.2

**Review by onedr0p — CHANGES_REQUESTED** [review]

File: charts/home-assistant/Chart.yaml (hunk 1 above)
Comment (angryninja48) [reply charts/home-assistant/Chart.yaml]: Done.

**Review by onedr0p — APPROVED** [review]

**Comment by onedr0p** [comment]
Thanks for the PR, I am sure this will help other people out as well 👍

**Pull request merged**